examples/material_gallery/lib/gallery/example_code_parser.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter/material.dart';
+
+const String _kStartTag = '// START ';
+const String _kEndTag = '// END';
+
+Map<String, String> _exampleCode;
```

Comment:
It's lazily initialized as I need the bundle to find the resources. I use the null value to detect if it is actually loaded and parsed, so I don't have to load it multiple times.